src/frontend/js/login.js

```diff
@@ -149,11 +217,38 @@ div:has(#submit_btn) {
     this.password = inputEl.value;
   }
 
+  checkLogin(e) {
+    if (e.key == "Enter") this.loginConfirm();
```

Comment:
In questo tipo di condizioni meglio usare la strict equality (https://www.guru99.com/difference-equality-strict-operator-javascript.html)

Reply:
messa la strict equality